README.md

```diff
@@ -99,3 +99,13 @@ generate a report, run using `--info`:
 ```shell
 ./gradlew sonarqube --info
 ```
+## Common Errors:
+If you see the following error when running `./gradlew build`:
+```
+./gradlew build
+> Task :buildSrc:validatePlugins FAILED
+
+FAILURE: Build failed with an exception.
+```
+
+You should try installing gradle using the command `brew install gradle` for MacOS, and then run via `gradle build`
```

Comment:
I think I am going to remove this section, because I am not confident enough that it wasn't related to some of the other things we ran into while setting up my env.